xarray/core/variable.py

```diff
@@ -993,6 +993,32 @@ def chunk(self, chunks=None, name=None, lock=False):
 
         return type(self)(self.dims, data, self._attrs, self._encoding, fastpath=True)
 
+    def _as_sparse(self, sparse_format=_default, fill_value=dtypes.NA):
```

Comment:
Currently, this is a private method.
Probably we can expose it to the public and add the same method to DataArray and Dataset as well in the future.